code/game/objects/items/tools/powertools.dm

```diff
@@ -40,7 +40,7 @@
 /obj/item/powertool/proc/on_transform(obj/item/source, mob/user, active)
 	SIGNAL_HANDLER
 
-	tool_behaviour = (active ? tool_act_off : tool_act_on)
+	tool_behaviour = (active ? tool_act_on : tool_act_off)
 	if(user)
 		balloon_alert(user, "attached [active ? "[action_off]" : "[action_on]"]")
```

Comment:
```suggestion
		balloon_alert(user, "attached [active ? "[action_on]" : "[action_off]"]")
```

This also seems to be the wrong way around

Reply:
confirmed that these are working correctly via my own testing.